ydb/src/grpc_wrapper/raw_topic_service/client.rs

```diff
@@ -17,11 +18,23 @@ impl RawTopicClient {
         }
     }
 
-    pub async fn create_topic(&mut self, req: RawCreateTopicRequest) -> RawResult<RawCreateTopicResult> {
-        request_with_result!(
+    pub async fn create_topic(&mut self, req: RawCreateTopicRequest) -> RawResult<()> {
+        request_without_result!(
```

Comment:
if i try to use RawCreateTopicResult, https://github.com/ydb-platform/ydb-rs-sdk/blob/763f27cfdeaef432b6f822b202081bc4727196f7/ydb/src/grpc_wrapper/grpc.rs#L22 error gets thrown. I am not quite sure if I am do smth about it. My best guess is that happens cuz of result struct being empty

Reply:
Server does not sent result for the operation, it can be bug or optimization.

Use without result now is ok.